keyboards/levinson/keymaps/jyh/keymap.c

```diff
@@ -0,0 +1,232 @@
+#include "levinson.h"
```

Comment:
Please change this line to `#include QMK_KEYBOARD_H`, and then delete Lines 2 and 3. QMK_KEYBOARD_H automatically includes all three of these files.